src/main/java/service/Calculator.java

```diff
@@ -0,0 +1,54 @@
+package main.java.service;
+
+import main.java.domain.Command;
+
+import java.util.Stack;
+
+public class Calculator {
+
+    Operator PM;
+    Operator MD;
+
+    public Calculator() {}
+
+    public int calculate(Command command) {
+        PM = Operator.PLUS;
+        MD = Operator.MULTIPLY;
+
+        int result = 0;
+
+        int[] numberArr = command.numberArr;
+        Operator[] optArr = command.optArr;
+
+        Stack<Integer> st = new Stack<>();
```

Comment:
자바는 일반적으로 변수명을 줄이지 않습니다.

```suggestion
        Stack<Integer> stack = new Stack<>();
```